validation/src/test/scala/hmda/validation/engine/lar/validity/LarValidityEngineSpec.scala

```diff
@@ -16,7 +18,7 @@ class LarValidityEngineSpec extends WordSpec with MustMatchers with LarValidityE
       }
 
       lars.foreach { lar =>
-        checkValidity(lar) mustBe a[scalaz.Success[_]]
+        checkValidity(lar, ValidationContext(None)) mustBe a[scalaz.Success[_]]
```

Comment:
The other tests are returning `.isSuccess mustBe true`. We should probably choose one (I don't think this was introduced as part of this PR).